public/app/features/dashboard/state/DashboardModel.ts

```diff
@@ -471,6 +471,28 @@ export class DashboardModel implements TimeModel {
     }
   }
 
+  private removeNullValuesFromVariables(templating: { list: VariableModel[] }) {
+    if (!templating.list.length) {
+      return templating;
+    }
+
+    for (const variable of templating.list) {
+      if (variable.current) {
+        // this is a safeguard for null value that breaks scenes dashboards.
+        //    expecting error at .includes(null) in order to not adjust
+        //    VariableOption type to avoid breaking changes
+        if (
+          variable.current.value === null ||
+          //@ts-expect-error
+          (Array.isArray(variable.current.value) && variable.current.value.includes(null))
+        ) {
+          variable.current = undefined;
```

Comment:
🤔 what happens when we assign undefined to `variable.current`? is it selecting the first available option?

Reply:
@axelavargas it will just be empty unless there are defaults further down the line.  This is all just to remove `null` from `variable.current.value` which break few places in scenes.